docs/source/optimizations/qscout/qscout_qss.ipynb

```diff
@@ -52,7 +52,9 @@
     "except ImportError:\n",
     "    print(\"Installing qiskit-superstaq...\")\n",
     "    %pip install -q qiskit-superstaq[examples]\n",
-    "    print(\"Installed qiskit-superstaq. Please restart the kernel to import the packages.\")\n",
```

Comment:
let's not remove this line. it's a bit misleading though. Basically you have to restart the kernel to grab the updated environment (with the new package installed) importing without restarting won't properly import the new package. Sometimes it works, sometimes it does not and restarting the kernel guarantees the newly installed package is picked up on [more info here](https://saturncloud.io/blog/solving-the-issue-installing-a-pip-package-from-within-a-jupyter-notebook-not-working/#:~:text=To%20resolve%20this%20issue%2C%20you,installed%20package%20without%20any%20issues.).

Reply:
Offline discussion: `%pip install ...` already prints something about the kernel, so we're fine.